src/codegen/utilities/interpreter_helpers.py

```diff
@@ -182,7 +187,7 @@ def generate_interpreter_function_call_args(function_metadata):
             else:
                 function_call_args.append(param.parameter_name)
 
-    return function_call_args
+    return function_call_args, dateTime_args
```

Comment:
I don't like this return value because it's overly specific and not really necessary.